txbits/app/usertrust/UserTrustModel.scala

```diff
@@ -22,16 +22,21 @@ import java.sql.Timestamp
 import play.api.Play.current
 import models.Withdrawal
 import org.joda.time.DateTime
+import anorm._
 
 class UserTrustModel(val db: String = "default") {
   def getTrustedActionRequests = DB.withConnection(db) { implicit c =>
-    SQLText.getTrustedActionRequests().map(row =>
+    SQL"""select email, is_signup from trusted_action_requests"""().map(row =>
       (row[String]("email"), row[Boolean]("is_signup"))
     ).toList
   }
 
   def getPendingWithdrawalRequests = DB.withConnection(db) { implicit c =>
-    SQLText.getPendingWithdrawalRequests().map(row =>
+    SQL"""select w.*, u.email, u.pgp, wc.address as destination from withdrawals w
+       left join users u on w.user_id = u.id
```

Comment:
Should this be an `inner join` since email is parsed as a String, not an Option[String]?